{
  "uid" : "bc80593bb1403b68",
  "name" : "test_02_login[xiaoma@126.com--请填写用户名和密码]",
  "fullName" : "test.test_web.test_login.Test_Login#test_02_login",
  "historyId" : "86e1c9a62ca26f5e064ac1639ce0984c",
  "time" : {
    "start" : 1668954782567,
    "stop" : 1668954786741,
    "duration" : 4174
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: assert 0",
  "statusTrace" : "self = <test.test_web.test_login.Test_Login object at 0x000001DDFE5C6B80>\nusername = 'xiaoma@126.com', password = '', expected = '请填写用户名和密码'\n\n    @pytest.mark.parametrize(\"username, password, expected\",\n                             testcase[\"c_example\"])\n    def test_02_login(self, username, password, expected):\n        self.home_proxy.to_login()\n        time.sleep(2)\n        self.login_proxy.enter(username, password)\n        time.sleep(2)\n        loginErr_text = self.login_proxy.get_loginErr()\n        assert loginErr_text == expected\n>       assert 0\nE       assert 0\n\ntest\\test_web\\test_login.py:50: AssertionError",
  "flaky" : false,
  "beforeStages" : [ {
    "name" : "_xunit_setup_method_fixture_Test_Login",
    "time" : {
      "start" : 1668954781317,
      "stop" : 1668954782566,
      "duration" : 1249
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: assert 0",
    "statusTrace" : "self = <test.test_web.test_login.Test_Login object at 0x000001DDFE5C6B80>\nusername = 'xiaoma@126.com', password = '', expected = '请填写用户名和密码'\n\n    @pytest.mark.parametrize(\"username, password, expected\",\n                             testcase[\"c_example\"])\n    def test_02_login(self, username, password, expected):\n        self.home_proxy.to_login()\n        time.sleep(2)\n        self.login_proxy.enter(username, password)\n        time.sleep(2)\n        loginErr_text = self.login_proxy.get_loginErr()\n        assert loginErr_text == expected\n>       assert 0\nE       assert 0\n\ntest\\test_web\\test_login.py:50: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "f001ff2e6aa50cb1",
      "name" : "stdout",
      "source" : "f001ff2e6aa50cb1.txt",
      "type" : "text/plain",
      "size" : 78
    } ],
    "parameters" : [ ],
    "hasContent" : true,
    "stepsCount" : 0,
    "shouldDisplayMessage" : true,
    "attachmentsCount" : 1
  },
  "afterStages" : [ {
    "name" : "_xunit_setup_method_fixture_Test_Login::0",
    "time" : {
      "start" : 1668954786744,
      "stop" : 1668954788814,
      "duration" : 2070
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "hasContent" : false,
    "stepsCount" : 0,
    "shouldDisplayMessage" : false,
    "attachmentsCount" : 0
  } ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "test.test_web"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login"
  }, {
    "name" : "host",
    "value" : "xiaoma"
  }, {
    "name" : "thread",
    "value" : "16956-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "test.test_web.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "expected",
    "value" : "'请填写用户名和密码'"
  }, {
    "name" : "password",
    "value" : "''"
  }, {
    "name" : "username",
    "value" : "'xiaoma@126.com'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "bc80593bb1403b68.json",
  "parameterValues" : [ "'请填写用户名和密码'", "''", "'xiaoma@126.com'" ]
}